docs/reference/api/docker_remote_api_v1.25.md

```diff
@@ -3211,7 +3225,9 @@ interactive session with the `exec` command.
     HTTP/1.1 200 OK
     Content-Type: application/vnd.docker.raw-stream
 
+    {% raw %}
```

Comment:
IIUC, the `{% raw %}` is to prevent Jekyll from trying to parse `{{ STREAM }}` as an instruction / template. Guess Jekyll is not as smart as Hugo - does that answer your question @stevvooe ?